tests/main/snap-env/task.yaml

```diff
@@ -1,5 +1,13 @@
 summary: inspect all the set environment variables prefixed with SNAP_ and XDG_
 
+details: |
+    Environment variables are widely used across Linux to provide convenient access to system
+    and application properties. Snapd uses specific environment variables to support running
+    snaps.
+
+    This test verifies that all the environment variables sued by snapd start with
+    SNAP_, XDG_ and EXTRA_.
```

Comment:
I think the part about extra is misleading and wrong.

Reply:
perhaps, because it is very old the test case? I don't see any env var with EXTRA_ nowadays but the test is checking for that

Reply:
I see 

tests/main/snap-env: extend to cover parallel installations of snaps

Add parallel installs variant of the test

Reply:
This was added 6 years ago

tests/main/snap-env: extend to cover parallel installations of snaps

Add parallel installs variant of the test

Reply:
Yes, but the wording implies that EXTRA is somehow something snapd manages. It is all-right, we can iterate later. Lots of tests to improve.